CIP-0006/README.md

```diff
@@ -1,20 +1,25 @@
 ---
 CIP: 6
 Title: Stake Pool Extended Metadata
-Authors: Markus Gufler <gufmar@gmail.com>, Mike Fullman <mike@fullman.net>
-Comments-URI: https://github.com/cardano-foundation/CIPs/pull/15
-Status: Draft
-Type: Standards
+Status: Active
+Category: Tools
```

Comment:
```suggestion
Category: Metadata
```
Happy for it to be either 🤓

Reply:
I'm glad you're happy with `Tools` then 🤓 since I already thought about which of these choices to make.  For other other CIPs tagged `Metadata` that metadata, even when off-chain, is associated with the chain somehow & in this case it isn't.

Also the target is a broad spectrum of operational procedures and tools as summarised best in this sentence (which itself could be a good description of the `Tools` category):
> Many additional attributes can be envisioned for future wallets, pool explorers, and information aggregators.

Reply:
> For other CIPs tagged Metadata, their metadata - even when off-chain - is associated with the chain somehow, and in this case it isn't.

Makes sense to me! agreed.